pwa-devdocs/src/_data/peregrine.yml

```diff
@@ -6,9 +6,6 @@ entries:
     - label: Routing in PWA Studio
       url: /peregrine/routing/
 
-    - label: Storybook UI testing
```

Comment:
@jcalcaben I removed the docs references for storybook but I think they aren't specific to just peregrine. Maybe we can migrate them to the `venia-concept` docs? LMK if you want me to do that and I can do it here, or you are welcome to push to my branch.

Reply:
If we're still going to use Storybook, we should move it somewhere else. If possible, can you move the `/peregrine/testing/storybook` directory to `pwa-devdocs/drafts` instead of deleting it so it doesn't get published. 

Then we can create another issue to revise the content and move it to a more appropriate place. That way that unrelated task won't hold up this issue.